.github/workflows/client-credential-benchmark.yml

```diff
@@ -0,0 +1,46 @@
+name: msal-node client-credential Regression Test
+on:
+    # Allows manual triggering of workflow
+    workflow_dispatch:
+
+    pull_request:
+        types: [opened, reopened, synchronize, ready_for_review]
+        paths:
+            - "regression-tests/msal-node/client-credential/*"
+            - "lib/msal-node/**/*"
+            - "lib/msal-common/**/*"
+            - "!**.md"
+            - ".github/workflows/client-credential.yml"
+            
+    merge_group:
+        types: [checks_requested]
+
+jobs:
+  benchmark:
+    name: Run msal-node client-credential Regression Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - name: Run benchmark
+        run: cd regression-tests/msal-node/client-credential && npm install && node index.js | tee output.txt
+
+      - name: Download previous benchmark data
+        uses: actions/cache@v3
+        with:
+          path: ./cache
+          key: ${{ runner.os }}-benchmark
+
+      - name: Store benchmark result
+        uses: benchmark-action/github-action-benchmark@v1
+        with:
+          name: msal-node client-credential Regression Test
+          tool: 'benchmarkjs'
+          output-file-path: regression-tests/msal-node/client-credential/output.txt
+          external-data-json-path: ./cache/client-credential-benchmark-data.json
```

Comment:
FWIW, although this `external-data-json-path` and the `actions/cache@v3` above were [mentioned in Continuous Benchmark's README](https://github.com/marketplace/actions/continuous-benchmark#minimal-setup), its [actual github workflow yml file](https://github.com/benchmark-action/github-action-benchmark/blob/master/.github/workflows/benchmarkjs.yml) did not need to use such an external data file. I ended up not using this setting.